views/index.hbs

```diff
@@ -36,7 +36,9 @@
         {{#if prs}}
             <div class="row">
                 <div class="large-12 columns">
-                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober</h2>
+                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober.</h2>
```

Comment:
I'm thinking we get rid of the 'pull requests submitted in Hacktober' bit now, what you think?